services/server/server.go

```diff
@@ -611,6 +618,49 @@ func (s *Server) ChangeSettings(ctx context.Context, req *serverpb.ChangeSetting
 	}, nil
 }
 
+// deleteAllAlertRules deletes all alerts present rules.
+// Returns nil in case of success, otherwise an error.
+func (s *Server) deleteAllAlertRules() error {
+	alertRuleIds := s.getAlertRuleIds()
+
+	if len(alertRuleIds) > 0 {
+		err := s.db.InTransaction(func(tx *reform.TX) error {
+			return models.RemoveRules(tx.Querier, alertRuleIds)
+		})
+		if err != nil {
+			return err
+		}
+
+		s.vmalert.RequestConfigurationUpdate()
+	}
+
+	return nil
+}
+
+// getAlertRuleIds lists the alert rules ids into an array.
+// Error is not returned, when occurres an empty array is returned.
+func (s *Server) getAlertRuleIds() []string {
+	var rules []*models.Rule
+	err := s.db.InTransaction(func(tx *reform.TX) error {
```

Comment:
why you are not returning this error?

Reply:
Do we have any plans with the error code? I can add it there, but as it is now, it will return an empty string in case of error which means it will not do anything.
Shall I add it?

Reply:
Does not exist anymore.